downloader/history_downloader/history_manager.py

```diff
@@ -0,0 +1,19 @@
+import ccxt
+
+
+class HistoryDownloader:
+
+    def __init__(self, exchange_name: str, crypto_symbol: str):

```

Comment:
Try to read about enums in python, maybe it will be better to use them in such situation when we know which `exchange_name's` or `crypto_sumbol's` are available upfront?

Reply:
Hmm, yes but now it is only for testing purposes. In the final versions when there is UI, the user will be able to pick only exchanges from the set, but that's only my opinion.

Reply:
We will be creating our own REST API for this, so backend should be ready anyway ;)